dissect/target/loaders/dir.py

```diff
@@ -15,15 +16,24 @@
 class DirLoader(Loader):
     """Load a directory as a filesystem."""
 
+    PREFIXES = ["", "fs"]
+
+    @classmethod
+    @cache
+    def _find_entry_path(cls, path: Path):
+        found = None
+        for prefix in cls.PREFIXES:
+            if find_dirs(path / prefix)[0] is not None:
+                found = prefix
+                break
+        return found
```

Comment:
```suggestion
        for prefix in cls.PREFIXES:
            if find_dirs(path / prefix)[0] is not None:
                return prefix
```

Reply:
I always try to avoid multiple returns in one function, but this is more compact. Trade-off I guess.